storage_service/locations/models/dspace.py

```diff
@@ -365,39 +365,39 @@ def _set_permissions(self, package):
         params = {'expand': 'bitstreams'}
         try:
             response = requests.get(url, headers=headers, params=params)
+            item = response.json()
         except Exception:
             LOGGER.warning('Error fetching bitstream information for handle %s', handle, exc_info=True)
         LOGGER.debug('REST API handle mapping %s %s', response.status_code, response)
         LOGGER.debug('Body %s', response.json())
 
         # Update bitstream policies & descriptions through REST API
-        for bitstream in response.json()['bitstreams']:
+        for bitstream in item['bitstreams']:
             url = dspace_url + bitstream['link']
             LOGGER.debug('Bitstream policy URL %s', url)
             body = bitstream
-            if bitstream['name'] == 'metadata.7z':
+            if bitstream['name'] in ['metadata.7z', 'metadata.zip']:
                 # Overwrite existing policies, instead of adding
                 body['policies'] = self.metadata_policy
                 # Add bitstream description for metadata when depositing to DSpace
-                body['description'] = 'Administrative information.'
-            elif bitstream['name'] == 'objects.7z':
+                body['description'] = 'Administrative information. Access restricted to Bentley staff.'
```

Comment:
Please, remove anything specific to Bentley for the pull request in the Archivematica repository.